cli/exec/exec.go

```diff
@@ -86,207 +86,207 @@ var Command = cli.Command{
 		cli.StringFlag{
+			EnvVar: "WOODPECKER_COMMIT_SHA",
 			Name:   "commit-sha",
-			EnvVar: "DRONE_COMMIT_SHA,WOODPECKER_COMMIT_SHA",
 		},
 		cli.StringFlag{
+			EnvVar: "WOODPECKER_COMMIT_REF",
 			Name:   "commit-ref",
-			EnvVar: "DRONE_COMMIT_REF,WOODPECKER_COMMIT_REF",
 		},
 		cli.StringFlag{
+			EnvVar: "WOODPECKER_COMMIT_REFSPEC",
 			Name:   "commit-refspec",
-			EnvVar: "DRONE_COMMIT_REFSPEC,WOODPECKER_COMMIT_REFSPEC",
 		},
 		cli.StringFlag{
+			EnvVar: "WOODPECKER_COMMIT_BRANCH",
 			Name:   "commit-branch",
-			EnvVar: "DRONE_COMMIT_BRANCH,WOODPECKER_COMMIT_BRANCH",
 		},
 		cli.StringFlag{
+			EnvVar: "WOODPECKER_COMMIT_MESSAGE",
 			Name:   "commit-message",
-			EnvVar: "DRONE_COMMIT_MESSAGE,WOODPECKER_COMMIT_MESSAGE",
 		},
 		cli.StringFlag{
+			EnvVar: "WOODPECKER_COMMIT_AUTHOR_NAME",
 			Name:   "commit-author-name",
-			EnvVar: "DRONE_COMMIT_AUTHOR_NAME,WOODPECKER_COMMIT_AUTHOR_NAME",
 		},
 		cli.StringFlag{
+			EnvVar: "WOODPECKER_COMMIT_AUTHOR_AVATAR",
 			Name:   "commit-author-avatar",
-			EnvVar: "DRONE_COMMIT_AUTHOR_AVATAR,WOODPECKER_COMMIT_AUTHOR_AVATAR",
 		},
 		cli.StringFlag{
+			EnvVar: "WOODPECKER_COMMIT_AUTHOR_EMAIL",
 			Name:   "commit-author-email",
-			EnvVar: "DRONE_COMMIT_AUTHOR_EMAIL,WOODPECKER_COMMIT_AUTHOR_EMAIL",
 		},
 		cli.IntFlag{
+			EnvVar: "WOODPECKER_PREV_BUILD_NUMBER",
 			Name:   "prev-build-number",
-			EnvVar: "DRONE_PREV_BUILD_NUMBER,WOODPECKER_PREV_BUILD_NUMBER",
 		},
 		cli.Int64Flag{
+			EnvVar: "WOODPECKER_PREV_BUILD_CREATED",
 			Name:   "prev-build-created",
-			EnvVar: "DRONE_PREV_BUILD_CREATED,WOODPECKER_PREV_BUILD_CREATED",
 		},
 		cli.Int64Flag{
+			EnvVar: "WOODPECKER_PREV_BUILD_STARTED",
 			Name:   "prev-build-started",
-			EnvVar: "DRONE_PREV_BUILD_STARTED,WOODPECKER_PREV_BUILD_STARTED",
 		},
 		cli.Int64Flag{
+			EnvVar: "WOODPECKER_PREV_BUILD_FINISHED",
 			Name:   "prev-build-finished",
-			EnvVar: "DRONE_PREV_BUILD_FINISHED,WOODPECKER_PREV_BUILD_FINISHED",
 		},
 		cli.StringFlag{
+			EnvVar: "WOODPECKER_PREV_BUILD_STATUS",
 			Name:   "prev-build-status",
-			EnvVar: "DRONE_PREV_BUILD_STATUS,WOODPECKER_PREV_BUILD_STATUS",
 		},
 		cli.StringFlag{
+			EnvVar: "WOODPECKER_PREV_BUILD_EVENT",
 			Name:   "prev-build-event",
-			EnvVar: "DRONE_PREV_BUILD_EVENT,WOODPECKER_PREV_BUILD_EVENT",
 		},
 		cli.StringFlag{
+			EnvVar: "WOODPECKER_PREV_BUILD_LINK",
 			Name:   "prev-build-link",
-			EnvVar: "DRONE_PREV_BUILD_LINK,WOODPECKER_PREV_BUILD_LINK",
 		},
 		cli.StringFlag{
+			EnvVar: "WOODPECKER_PREV_COMMIT_SHA",
 			Name:   "prev-commit-sha",
-			EnvVar: "DRONE_PREV_COMMIT_SHA,WOODPECKER_PREV_COMMIT_SHA",
 		},
 		cli.StringFlag{
+			EnvVar: "WOODPECKER_PREV_COMMIT_REF",
 			Name:   "prev-commit-ref",
-			EnvVar: "DRONE_PREV_COMMIT_REF,WOODPECKER_PREV_COMMIT_REF",
 		},
 		cli.StringFlag{
+			EnvVar: "WOODPECKER_PREV_COMMIT_REFSPEC",
 			Name:   "prev-commit-refspec",
-			EnvVar: "DRONE_PREV_COMMIT_REFSPEC,WOODPECKER_PREV_COMMIT_REFSPEC",
 		},
 		cli.StringFlag{
```

Comment:
own flag.go file?

Reply:
Would be nice and would match how it has been done for server and agent.